src/operators.jl

```diff
@@ -531,3 +531,32 @@ function Base.:*(
 )
     return x * A
 end
+
+function Base.complex(
+    r::Union{
+        GenericVariableRef{<:Real},
+        GenericAffExpr{<:Real},
+        GenericQuadExpr{<:Real},
+    },
+    i::Real,
+)
+    return r + im * i
+end
+
+function Base.complex(
+    r::Real,
+    i::Union{
+        GenericVariableRef{<:Real},
+        GenericAffExpr{<:Real},
+        GenericQuadExpr{<:Real},
+    },
+)
+    return r + im * i
+end
+
+function Base.complex(
+    r::Union{GenericVariableRef{<:Real},GenericAffExpr{<:Real}},
+    i::Union{GenericVariableRef{<:Real},GenericAffExpr{<:Real}},
+)
```

Comment:
Why are you not allowing `GenericQuadExpr{<:Real}`here?

Reply:
Because I can't copy-paste correctly haha (and then I added tests for the methods that exist, not thinking)